scss/style.scss

```
@@ -0,0 +1,533 @@
+    color: #ff6363;
+    &:hover {
+      color: darken($color: #ff6363, $amount: 25%);
+    }
+  }
+  .icon-thumbs-up {
+    color: #00a2ff;
+    &:hover {
+      color: darken($color: #00a2ff, $amount: 25%);
+    }
+  }
+}
+.spain {
+  font-family: $josefin;
+  padding: 220px 0 120px;
+  &--left {
+    @include back();
+    background-image: url(../img/BG3.jpg);
+  }
+  &--right {
+    @include back();
+    background-image: url(../img/spain.jpg);
+    .row {
+      justify-content: flex-end;
+    }
+  }
+  .body__wrapper {
+    background: rgba(35, 35, 35, 0.9);
+    padding: 44px 32px;
+  }
+  &__subheader,
+  &__header {
+    font-size: 20px;
+    line-height: 28px;
+  }
+  &__subheader {
+    font-weight: 300;
+    letter-spacing: 0.1em;
+    margin-bottom: 16px;
+  }
+  &__header {
+    font-weight: normal;
+    text-transform: uppercase;
+    margin-bottom: 16px;
+  }
+  &__info {
+    font-size: 16px;
+    line-height: 28px;
+    letter-spacing: 0.1em;
+    margin-bottom: 24px;
+  }
+  &__button {
+    @include but();
+  }
+}
+.feedback {
+  @include back();
+  background-image: url(../img/BGlast.jpg);
+  padding: 120px 0 180px;
+  .row > * {
+    padding: 0;
+  }
+  &__img {
+    height: 100%;
+    font-size: 0;
+    line-height: 0;
+  }
+  &__header {
+    @include h2();
+    text-align: center;
+    margin-bottom: 60px;
+  }
+  &__subheader {
+    font-family: $josefin;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    text-transform: uppercase;
+    margin-bottom: 10px;
+  }
+  &__info {
+    font-family: $josefin;
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 28px;
+    letter-spacing: 0.1em;
+  }
+  body .container &__wrapper {
+    background-color: #333333;
+    padding: 48px 60px;
+    .icon-quote-left {
+      display: inline-block;
+      font-size: 20px;
+      color: $blue;
+      vertical-align: super;
+    }
+  }
+}
+.footer {
+  text-align: center;
+  background-color: #000;
+  padding: 80px 0;
+  font-family: $josefin;
+  &__header {
+    &__link {
+      text-decoration: none;
+      font-weight: bold;
+      font-size: 24px;
+      line-height: 28px;
+      letter-spacing: 0.1em;
+      text-transform: uppercase;
+      color: $white;
+    }
+    margin-bottom: 8px;
+  }
+  &__info {
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2em;
+    margin-bottom: 20px;
+  }
+  i {
+    cursor: pointer;
+    font-size: 32px;
+    &:hover {
+      color: $blue;
+    }
+    &:not(:last-child) {
+      margin-right: 32px;
+    }
+  }
+}
+@import "adaptive";
+#nav-icon2 {
+  right: 42px;
+  top: 6px;
+  width: 60px;
+  height: 45px;
+  position: absolute;
+  -webkit-transform: rotate(0deg);
+  -moz-transform: rotate(0deg);
+  -o-transform: rotate(0deg);
+  transform: rotate(0deg);
+  -webkit-transition: 0.5s ease-in-out;
+  -moz-transition: 0.5s ease-in-out;
+  -o-transition: 0.5s ease-in-out;
+  transition: 0.5s ease-in-out;
```

```suggestion
  transform: rotate(0deg);
  transition: 0.5s ease-in-out;
```
не забирай хліб у автопрефіксера=)

---

По старій звичці)